Core/PrivacyConfiguration.swift

```diff
@@ -19,81 +19,146 @@
 
 import Foundation
 
-public struct PrivacyConfiguration: Codable {
-    public typealias FeatureName = String
-    public typealias UnprotectedList = [ExceptionEntry]
-    
-    public let features: [FeatureName: PrivacyFeature]
-    public let unprotectedTemporary: UnprotectedList
+public protocol PrivacyConfiguration {
+
+    /// Identifier of given Privacy Configuration, typically an ETag
+    var identifier: String { get }
```

Comment:
Typically? What else can this be and when will it not be an etag?

Reply:
It can be anything as it's a protocol. But for default (non-tests) implementation it's an ETag.

Reply:
But if you want to make these docs more clear feel free to suggest anything :)